homeassistant/components/proxmox/binary_sensor.py

```diff
@@ -0,0 +1,54 @@
+import homeassistant.components.proxmox as proxmox
+from homeassistant.components.switch import SwitchDevice
+
+DOMAIN = 'proxmox'
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up Proxmox VE binary sensor."""
+    nodes = hass.data[proxmox.DATA_PROXMOX_NODES]
+    sensors = []
+
+    for node_name in nodes.keys():
+        item = nodes[node_name]
+        if 'type' not in item or item['type'] != 'node':
+            sensors.append(PXMXBinarySensor(hass, node_name, 'Is Running', lambda node: 'running' == node['status']))
+
+    async_add_entities(sensors)
+
+
+class PXMXBinarySensor(SwitchDevice):
```

Comment:
See above. Also, the docstring is missing.

Reply:
Fixed.